app/models/manageiq/providers/embedded_ansible/automation_manager/job.rb

```diff
@@ -108,14 +108,16 @@ def update_with_provider_object(raw_job)
   end
 
   def raw_stdout_json
-    miq_task.context_data[:ansible_runner_stdout]
+    miq_task.try(&:context_data).try(:[], :ansible_runner_stdout) || []
```

Comment:
Doesn't `more_core_ext` have a `deep_fetch` or something that would handle this without all of the `.try` calls?

Reply:
unforunately deep_fetch doesn't work on nil, so you have to have a preliminary object...in this case since the chain "starts" with a possibility of nil, I can't use it.  At best, I could do

```ruby
miq_task.try(&:context_data).try(:deep_fetch, :ansible_runner_stdout) || []
```

which is about the same if not worse ☹️